sr_description/hand/xacro/thumb/thdistal.urdf.xacro

```diff
@@ -100,20 +100,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
           <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0"
           izz="0" />
         </inertial>
-        <visual>
+        <!--visual>
```

Comment:
I guess there was a problem in the chain that had two links attached to the middle. now tip is attached to dummy distal. I just know that the generic srdf I created was taking into account the biotac specificities mainly due to thmiddle being called thbiotac, but that is fixed now. You should double check if all the collision pairs are nicely generated with this new link arrangement. I documented a lot the generator srdf xacros, and I had to use a lot of macros for far away, near and nearby fingers as they touch or not depending on the distances.